### ISSUE TITLE
Homing with corexy on grbl 0.9i

### ISSUE THREAD

**Issue by Faisalbelwadi** [grbl/grbl#1248]
I have made a drawing machine which is a corexy machine wih 2 axis and servo support
Having grbl 0.9i from

https://github.com/misan/grbl-servo

I have setup limit switches and am pretty sure the hardware is perfect as hard limits are triggered on both x and y successfully

BUT when i run a homing cycle, both x and y start to move good, but then when one of the switches is triggered, the second axis doesn't move (or moves vert very slow) and does not home and makes a strong humming sound and homing fails

Note: if i deliberately press the switch while humming, there is no homing fail.
In short only one axis gets homed properly and the second tries to home, but cannot.

Can you provide me help with this matter as I've searched all over the internet but nothing worked

**Comment by chamnit** [comment]
Once again, this is not a supported version of Grbl. Closing. Please stop posting multiple times and emailing me personally.
